crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/validation/service/MeansAssessmentValidationServiceTest.java

```diff
@@ -160,4 +166,35 @@ public void givenNoRepId_whenIsOutstandingAssessmentIsInvoked_thenFalseIsReturne
         requestDTO.setRepId(null);
         assertThat(meansAssessmentValidationService.isOutstandingAssessment(requestDTO)).isFalse();
     }
+    @Test
+    public void givenNoTimeStamp_whenIsFinAssessmentModifiedByOtherUserInvoked_thenTrueIsReturned() {
+        MeansAssessmentRequestDTO requestDTO = TestModelDataBuilder.getMeansAssessmentRequestDTO(true);
+        when(maatCourtDataService.getFinancialAssessment(any(),any())).thenReturn(TestModelDataBuilder.getFinancialAssessmentDTO());
+        assertThat(meansAssessmentValidationService.isAssessmentModifiedByAnotherUser(requestDTO)).isTrue();
+    }
+
+    @Test
+    public void givenAValidTimeStampInRequest_whenMisMatchTimeStampInFinancialAssessmentDTO_thenTrueIsReturned() {
```

Comment:
Method naming convention seems a bit wonky here. I would suggest using the following pattern to ensure consistency:

`givenMisMatchedTimeStamp_whenIsAssessmentModifiedByAnotherUserIsInvoked_thenTrueIsReturned`

Reply:
Fixed now